src/platform/qt/PaletteView.h

```diff
@@ -13,8 +13,6 @@
 
 #include "ui_PaletteView.h"
 
-class GameController;
```

Comment:
Uh, you include GameController.h on line 11 so this is redundant and it causes a compiler error?